### PR TITLE
doc: archive doc about adding new tf/dcl resources

### PR DESCRIPTION
**Pull request by yuwenma**
- add an announcement 
- move the files about adding TF/DCL resources to ARCHIVE dir.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+### ** New Config Connector is coming in 2024 Q4 **
+
+We are developing a faster, more Kubernetes native and more reliable Config Connector reconciliation approach(Direct Controller).
+Thus, we decide to hold off any new PRs about resource coverage until mid Q4 2024.
```
Comment (cheftako) [inline README.md:4]: For resources which are Beta *and* are currently of type TF I think we should probably continue supporting new PRs.

Comment (yuwenma) [reply README.md]: Updated.

File: README.md [hunk 2]
```diff
@@ -33,3 +40,4 @@ For simple starter examples, see the
 ## Contributing to Config Connector
 
 Please refer to our [contribution guide](CONTRIBUTING.md) for more details.
```
Comment (justinsb) [inline README.md:42]: Is this link broken now?

Comment (yuwenma) [reply README.md:42]: Good catch. I put back the CONTRIBUTION.md. I think it does not contain much TF/DCL specific content

**Review by justinsb — APPROVED** [review]

**Pull request merged**